### PR TITLE
PHPDocs cleans ups

### PR DESCRIPTION
**Pull request by SpacePossum**
And some tabs versus spaces.

### CODE REVIEW

File: src/Silex/ServiceProviderInterface.php [hunk 1]
```diff
@@ -23,6 +23,8 @@
      *
      * This method should only be used to configure services and parameters.
      * It should not get services.
+     *
+     * @param Application $app
```
Comment (fabpot) [inline src/Silex/ServiceProviderInterface.php]: This should be removed as it does not bring anything useful.

File: src/Silex/ServiceProviderInterface.php [hunk 2]
```diff
@@ -32,6 +34,8 @@ public function register(Application $app);
      * This method is called after all services are registered
      * and should be used for "dynamic" configuration (whenever
      * a service must be requested).
+     *
+     * @param Application $app
```
Comment (fabpot) [inline src/Silex/ServiceProviderInterface.php]: Same as above

**Comment by fabpot** [comment]
Looks good to me apart from the 2 comments I've just made. Also, can you rebase the PR? (rebasing will remove the merge commit that we don't want when merging). Thanks.

**Comment by SpacePossum** [comment]
Sure thing :) 
I've removed the two lines from the PR, rebased and commits squash'ed.

**Comment by fabpot** [comment]
Thank you @SpacePossum.

**Pull request merged**